### PR TITLE
Request to release OpenTelemetry.Instrumentation.AWS package

### PR DESCRIPTION
**Pull request by ssaraswati**
## Changes

Requesting to release OpenTelemetry.Instrumentation.AWS. I would like to test the changes made in #1572 on a AWS .NET SDK project I am working on.

@open-telemetry/dotnet-contrib-maintainers

### CODE REVIEW

**Comment by Kielek** [comment]
@ppittle, @srprash, are you fine with this? If so, I can handle the process.

File: src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog - OpenTelemetry.Instrumentation.AWS
 
-## Unreleased
+## 1.1.0-beta.4
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md:3]: Please keep Unreleased section, even if it is emoty.

```suggestion
## Unreleased

## 1.1.0-beta.4
```

**Review by ppittle — APPROVED** [review]

**Comment by ppittle** [comment]
> @ppittle, @srprash, are you fine with this? If so, I can handle the process.

As long as this is a beta release, I'm fine with it.  Thanks @Kielek !

File: src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.1.0-beta.4
+
+Released 2024-Apr-11
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md:7]:
```suggestion
Released 2024-Apr-12
```

**Review by Kielek — APPROVED** [review]

**Comment by Kielek** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/releases/tag/Instrumentation.AWS-1.1.0-beta.4

**Pull request merged**